internals/models.py

```diff
@@ -953,19 +953,79 @@ def get_in_milestone(
     q = Feature.query()
     q = q.order(Feature.name)
     q = q.filter(Feature.shipped_milestone == milestone)
+    q = q.filter(Feature.impl_status_chrome.IN([ENABLED_BY_DEFAULT, DEPRECATED, REMOVED]))
```

Comment:
Hmm...  I get it. We should remove this condition. This will also remove the need for querying the legacy dev trials separately as features with no dt field set will also be included automatically. 

Also, as you said -  _'the feature is estimated to have impl_status_chrome == ENABLED_BY_DEFAULT by the time that that future milestone arrives'_, this makes me thinking that we will have to update each feature's  impl_status_chrome according to the future milestone as different feature types (ENABLED_BY_DEFAULT, DEPRECATED, REMOVED) all can have origin trials and dev trials. But when they are shipped, the field  `impl_status_chrome` will be different. Since the frontend uses `impl_status_chrome` for sorting according to shippingType, the API will have to return the correct status as per the milestone.